setup/sql/dbupdate_04.php

```diff
@@ -23916,7 +23916,7 @@ function writeCtrlClassEntry(ilPluginSlot $slot, array $plugin_data) {
 				$timestamp = $certificateFile->getMTime();
 
 				if (false !== $content) {
-					$backgroundImagePath = $relativePath . $relativePath . '/background.jpg';
```

Comment:
This should be already fixed: https://github.com/ILIAS-eLearning/ILIAS/commit/a8781a6912bcebc24a04bc6d61546e3333cfc312